File: demo_lottery/src/main/java/com/sheenhill/demo_lottery/MainActivity.java
package com.sheenhill.demo_lottery;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;

import android.os.Bundle;

import com.sheenhill.common.util.LogUtil;
import com.sheenhill.demo_lottery.databinding.ActivityMainBinding;
import com.sheenhill.demo_lottery.databinding.FragmentLotteryBinding;

/* 单独为app时，入口是这个activity */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        DataBindingUtil.setContentView(this,R.layout.fragment_lottery);
        LogUtil.i(BuildConfig.LOG_TEST);
    }
}

File: demo_lottery/src/main/java/com/sheenhill/demo_lottery/MainActivity.kt
package com.sheenhill.demo_lottery

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import androidx.databinding.ViewDataBinding
import com.sheenhill.common.activity.K_BaseActivity
import com.sheenhill.common.util.LogUtil

/* 单独为app时，入口是这个activity */
class MainActivity : K_BaseActivity(R.layout.activity_main){
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setLightStatusBarStyle()
        LogUtil.i(BuildConfig.LOG_TEST)
    }
}